config/environments/development.rb

```diff
@@ -64,10 +64,6 @@
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
-  # Use an evented file watcher to asynchronously detect changes in source code,
-  # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
So is this just tidying up to remove an error, that must have been around since 7 May! Rather than anything to do with the postgres upgrade? If so I'd be inclined to do that in a fresh branch and PR. Incase everything goes wonky and we end up reverting.

Reply:
Correct, it shows how much Content Data is worked on 😆  I was being a bit lazy including it in this PR so I'll split them into two.

Reply:
Done!